addons/spreadsheet/static/src/commands.json

```diff
@@ -243,86 +222,37 @@
       "row": 0,
       "id": "1",
       "definition": {
-          "metaData": {
-              "resModel": "crm.lead",
-              "columns": [
-                  "name",
-                  "contact_name",
-                  "email_from",
-                  "phone",
-                  "company_id",
-                  "user_id",
-                  "activity_ids",
-                  "my_activity_date_deadline",
-                  "expected_revenue",
-                  "recurring_revenue_monthly",
-                  "stage_id"
-              ]
-          },
-          "searchParams": {
-              "domain": [],
-              "context": {},
-              "orderBy": [
-                  {
-                      "name": "my_activity_date_deadline",
-                      "asc": true
-                  },
-                  {
-                      "name": "user_id",
-                      "asc": true
-                  }
-              ]
-          },
-          "name": "Pipeline by My Activity Deadline"
-      },
-      "dataSourceId": "4fc9a122-b55a-49be-b834-ec2af1cb7e02",
-      "linesNumber": 13,
-      "columns": [
-          {
-              "name": "name",
-              "type": "char"
-          },
-          {
-              "name": "contact_name",
-              "type": "char"
-          },
-          {
-              "name": "email_from",
-              "type": "char"
-          },
-          {
-              "name": "phone",
-              "type": "char"
-          },
-          {
-              "name": "company_id",
-              "type": "many2one"
-          },
-          {
-              "name": "user_id",
-              "type": "many2one"
-          },
-          {
-              "name": "activity_ids",
-              "type": "one2many"
-          },
-          {
-              "name": "my_activity_date_deadline",
-              "type": "date"
-          },
-          {
-              "name": "expected_revenue",
-              "type": "monetary"
-          },
+        "columns": [
+          "name",
+          "contact_name",
+          "email_from",
+          "phone",
+          "company_id",
+          "user_id",
+          "activity_ids",
+          "my_activity_date_deadline",
+          "expected_revenue",
+          "recurring_revenue_monthly",
+          "stage_id"
+        ],
+        "domain": [],
+        "model": "crm.lead",
+        "context": {},
+        "orderBy": [
```

Comment:
note that orderBy for list is an array while it's a single measure for pivots and charts

Reply:
Is it really possible to sort with two fields in a list ? I cannot do it from the UI (or I missed something)

Reply:
Yes you can, sort several columns one after the other.
While the UI doesn't really reflect it, "previous" sorts are still taken into account and becomes "secondary sort".
Check the search_read rpc ;)

Reply:
😬